src/coefficients/StdFunctionCoefficient.hpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+// # Author: Jonathan Wong @ LLNL.
+
+/**
+    @file StdFunctionCoefficient.hpp
+ */
+
+#ifndef STD_FUNCTION_COEFFICIENT_HPP
+#define STD_FUNCTION_COEFFICIENT_HPP
+#include <functional>
+#include "mfem.hpp"
+
+using namespace mfem;
+
+/**
+   \brief StdFunctionCoefficient is an easy way to make an mfem::Coefficient using a lambda
+
+   This is a place holder until the coefficient of the same name is merged into mfem.
+
+*/
+class StdFunctionCoefficient : public Coefficient
+{
+public:
+  /// Constructor that takes in an mfem Vector representing the coordinates and produces a double
+  StdFunctionCoefficient(std::function<double(mfem::Vector &)> func);
+  virtual ~StdFunctionCoefficient() {}
+
+  virtual double Eval (ElementTransformation &Tr, const IntegrationPoint &ip);
+private:
+  std::function<double(mfem::Vector &)> func_;
```

Comment:
Not sure i like having a trailing underscore to indicate its a function/callable object. If that is what you are doing.  Is this a convention you've seen before? Thoughts from the team?

Reply:
Someone working on another project started the convention instead of "m_"-prefix. This is what happens when I re-use code from other projects.